### PR TITLE
Fix/null default

### PR DESCRIPTION
**Pull request by AlexanderMann**
# Motivation
JSONSchema defines a `default` field to hint to validators, etc., that the given property does not need to meet it's `not null` constraint, and that if this failure case is reached, to simply use the default instead.

Presently, we do not handle a couple things:
- create a table with defaults
- our persist method doesn't play nice with default values in general
  - even if the column has a default set in SQL, it will not utilize that when the copy places `null`
  - this becomes especially tricky when you have mixed null/not-null values in a csv column...

## Notes
If there is a simple solution to get the CSV copy to play nicely, then I think we should use that.

This brute force approach works though.

## Suggested Musical Pairing
https://soundcloud.com/ugly-casanova/barnacles

### CODE REVIEW

File: target_postgres/json_schema.py [hunk 1]
```diff
@@ -101,12 +101,14 @@ def is_nullable(schema):
 
 
 def _helper_simplify(root_schema, child_schema):
+    ret_schema = {}
+
     if not child_schema:
-        return child_schema
+        pass
```
Comment (awm33) [inline target_postgres/json_schema.py]: Could the if just be removed here then? Does it really not affect thel ogic?

File: tests/fixtures.py [hunk 2]
```diff
@@ -36,6 +36,10 @@
             'name': {
                 'type': ['string']
             },
+            'paw_size': {
+                'type': ['integer'],
+                'default': 314159
```
Comment (awm33) [inline tests/fixtures.py]: That's a big paw

Comment (AlexanderMann) [reply tests/fixtures.py]: These cats have paws for pi and pi alone

File: target_postgres/json_schema.py (hunk 1 above)
Comment (AlexanderMann) [reply target_postgres/json_schema.py]: Actually, I think that case covered up another bug. Schemas are totally allowed to be completely empty. Those reference objects types. I'll add a better check for that.

File: tests/fixtures.py (hunk 2 above)
Comment (awm33) [reply tests/fixtures.py]: @AlexanderMann cat pi https://i.pinimg.com/originals/60/46/5d/60465da88007d8e5b28f265c7805abce.jpg

**Pull request merged**